frontend/src/Student/components/DisplayNameInput.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { Label, FormGroup, UncontrolledTooltip } from "reactstrap";
+import { Field } from "formik";
+import FontAwesomeIcon from "@fortawesome/react-fontawesome";
+import FaQuestionCircle from "@fortawesome/fontawesome-free-solid/faQuestionCircle";
+
+export default function DisplayNameInput({
+  values,
+  touched,
+  errors,
+  displayNameNeeded,
```

Comment:
It looks like the `displayNameNeeded` variable never gets used? Other than that I think this all looks good.

Reply:
Changed in newest conflict